### PR TITLE
Remove net452 and net46 from the in-memory exporter project

### PR DESCRIPTION
**Pull request by joaopgrassi**
Another project from #2138

* [x] `CHANGELOG.md` updated for non-trivial changes
* [x] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

**Review by CodeBlanch — APPROVED** [review]
LGTM

File: src/OpenTelemetry.Exporter.InMemory/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ please check the latest changes
 
 ## Unreleased
 
+* Removes support for .NET Framework 4.5.2 and 4.6. The minimum .NET Framework version
+  supported is .NET 4.6.1. [2138](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138)
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.InMemory/CHANGELOG.md:13]:
```suggestion
  supported is .NET 4.6.1. ([2138](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138))
```

**Review by reyang — APPROVED** [review]
LGTM.

**Comment by joaopgrassi** [comment]
I missed the `OpenTelemetry.Tests` using the InMemory project.. as these don't run on Linux :(. I guess we can leave this and merge once that one is also changed to remove these targets?

**Comment by cijothomas** [comment]
> I missed the `OpenTelemetry.Tests` using the InMemory project.. as these don't run on Linux :(. I guess we can leave this and merge once that one is also changed to remove these targets?

You can remove net452 and net46 from OpenTelemetry.Tests and instead add net461

**Comment by joaopgrassi** [comment]
> You can remove net452 and net46 from OpenTelemetry.Tests and instead add net461
Yep, done!

File: test/OpenTelemetry.Tests/Logs/LogRecordTest.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // </copyright>
 
-#if !NET452 && !NET46
+#if !NET461
```
Comment (cijothomas) [inline test/OpenTelemetry.Tests/Logs/LogRecordTest.cs:17]: we'll get rid of netcore2.1 soon, so this'd be lot more simpler.! (2.1 is EOL next month!) We can start removing as soon as the current wave of .netframework removals.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**